src/storage/notion.rs

```diff
@@ -187,9 +186,17 @@ impl Storage for NotionStorage {
                     .map(|opt| opt.unwrap())
                     .collect();
 
-                println!("Deleting Existing Blocks(Count: {}) using page archive method in single hit", block_ids.len());
+                println!(
+                    "Deleting Existing Blocks(Count: {}) using page archive method in single hit",
+                    block_ids.len()
+                );
                 // delete_page
-                match delete_blocks(&self.api_token, vec![BlockId::from_str(page_id.to_string().as_ref()).unwrap()]).await {
+                match delete_blocks(
+                    &self.api_token,
+                    vec![BlockId::from_str(page_id.to_string().as_ref()).unwrap()],
+                )
+                .await
+                {
                     Err(er) => eprintln!(
```

Comment:
how do we recover from this, user can't do much if it failed. How do we safegaurd what user has recorded?